src/commands/import.ts

```diff
@@ -1120,19 +1144,58 @@ command
 
       logger.info(`Created ${customFieldsToCreate.length} custom field(s)`);
 
-      logger.info('Checking if "Status" field is configured correctly...');
+      // At the time of writing this, the GraphQL mutation 'updateProjectV2Field' is only available on
+      // GitHub.com
+      const shouldConfigureStatusField = typeof gitHubEnterpriseServerVersion === 'undefined'
+      if (shouldConfigureStatusField) {
+        const sourceProjectStatusField = sourceProject.fields.nodes.find(field => field.name == "Status")!;
```

Comment:
We should use the strict equality operator here:

```suggestion
        const sourceProjectStatusField = sourceProject.fields.nodes.find(field => field.name === "Status")!;
```